1-mandatory/3-reading-list.js

```diff
@@ -47,6 +77,15 @@ As an example for this exercise, you might do the following steps
 - Modify the books so that they have the correct alreadyRead value
 - All tests should turn green!!
 **/
+function logBooks() {
```

Comment:
Here another error is indicated by the unit tests (again in the logs on the check tab).

This time the error is:
`
 SyntaxError: /home/runner/work/JavaScript-Core-2-Coursework-Week2/JavaScript-Core-2-Coursework-Week2/1-mandatory/3-reading-list.js: Identifier 'logBooks' has already been declared. (80:9)

      78 | - All tests should turn green!!
      79 | **/
    > 80 | function logBooks() {
         |          ^
      81 |   books.forEach((book) => {
      82 |     if (book.alreadyRead) {
      83 |       console.log("You've already read \"" + book.title + "\" by " + book.author);
`

Now it means you have two functions called logBooks. Defined on line 48 and 80. If you follow the instruction in this file, what the authors meant is to modify the original function on line 48 and not the write it again on line 80 (so there should be second function). This might be a bit ambiguous (the instructions in the task), but hopefully the check logs help.